psi4/src/psi4/psimrcc/mrcc_t2_amps.cc

```diff
@@ -42,6 +42,10 @@
 namespace psi {
 namespace psimrcc {
 
+// Many functions in this file have six-index local variables.
```

Comment:
wow

Reply:
I think these functions are never called. It must have been an attempt to implement Mk-MRCCSDT.